.swiftlint.yml

```diff
@@ -10,6 +10,7 @@ included:
   
 excluded:
   - Tests/LicensePlistTests/XcodeProjects
+  - Tests/LicensePlistTests/Entity/FileReader/XcodeProjectFileReaderTests.swift
```

Comment:
Assert texts in this file are very long, but they are not resource file. so I tried to ignore them.